### PR TITLE
px update

### PR DESCRIPTION
**Pull request by tusharrautOO7**
- Changed px default image to  : portworx/px-enterprise:2.0.3.7
- Changed px-lighthouse default image to : portworx/px-lighthouse:2.0.4
- Added advanced placement constraint options.

### CODE REVIEW

File: frameworks/portworx/universe/config.json [hunk 1]
```diff
@@ -107,9 +107,12 @@
           },
           "placement_constraint": {
             "title": "Placement constraint",
-            "description": "Marathon-style placement constraint for nodes. Example: 'hostname:UNIQUE'",
+            "description": "Marathon-style placement constraint for nodes. (e.g., [[\"hostname\", \"UNIQUE\"]]).",
             "type": "string",
-            "default": "hostname:UNIQUE"
+			"default": "[[\"hostname\", \"UNIQUE\"]]",
```
Comment (disrani-px) [inline frameworks/portworx/universe/config.json:112]: Can you confirm this doesn't break upgrades? ie define a constraint with the currently released framework and then update to this version

Comment (tusharrautOO7) [reply frameworks/portworx/universe/config.json:112]: Verified upgrade, working as expected.
Deployed portworx service with old version with placement constraints : "placement_constraint": "hostname:LIKE:(70.0.86.175|70.0.86.181|70.0.86.184)"
- Once the px is ready added new node into DCOS cluster and then updated portworx service using command: `dcos portworx update start --package-version=stub-universe`
- Observed px node came on same nodes.

File: frameworks/portworx/universe/config.json [hunk 2]
```diff
@@ -68,7 +68,7 @@
           "portworx_image": {
             "title": "Portworx Image name",
             "type": "string",
-            "default": "portworx/px-enterprise:2.0.3"
+            "default": "portworx/px-enterprise:2.0.3.7"
```
Comment (disrani-px) [inline frameworks/portworx/universe/config.json]: Use `2.0.3.8`

**Review by disrani-px — APPROVED** [review]

File: frameworks/portworx/universe/config.json (hunk 2 above)
Comment (tusharrautOO7) [reply frameworks/portworx/universe/config.json]: Changed.

File: frameworks/portworx/universe/config.json [hunk 3]
```diff
@@ -68,7 +68,7 @@
           "portworx_image": {
             "title": "Portworx Image name",
             "type": "string",
-            "default": "portworx/px-enterprise:2.0.3"
+            "default": "portworx/px-enterprise:2.0.3.8"
```
Comment (piyush-nimbalkar) [inline frameworks/portworx/universe/config.json:71]: We need to change the images in universe/resource.json as well.

**Review by piyush-nimbalkar — CHANGES_REQUESTED** [review]

File: frameworks/portworx/universe/config.json (hunk 3 above)
Comment (tusharrautOO7) [reply frameworks/portworx/universe/config.json:71]: Changed

**Pull request closed without merging**